src/org/scada_lts/dao/SynopticPanelDAO.java

```diff
@@ -0,0 +1,156 @@
+            + COLUMN_NAME_SP_VECTOR_IMAGE + ", "
+            + COLUMN_NAME_SP_COMPONENT_DATA + " "
+            + "from "
+            + TABLE_NAME;
+
+    private static final String SP_INSERT = ""
+            + "insert synopticPanels("
+            + COLUMN_NAME_SP_XID + ", "
+            + COLUMN_NAME_SP_NAME + ", "
+            + COLUMN_NAME_SP_VECTOR_IMAGE + ", "
+            + COLUMN_NAME_SP_COMPONENT_DATA + ") "
+            + "values (?,?,?,?)";
+
+    private static final String SP_UPDATE = ""
+            + "update synopticPanels set "
+            + COLUMN_NAME_SP_XID + "=?, "
+            + COLUMN_NAME_SP_NAME + "=?, "
+            + COLUMN_NAME_SP_VECTOR_IMAGE + "=?, "
+            + COLUMN_NAME_SP_COMPONENT_DATA + "=? "
+            + "where "
+            + COLUMN_NAME_SP_ID + "=?";
+
+    private static final String SP_DELETE = ""
+            + "delete from synopticPanels where "
+            + COLUMN_NAME_SP_ID + "=?";
+
+    @Override
+    public Object create(SynopticPanel entity) {
+        KeyHolder keyHolder = new GeneratedKeyHolder();
+        DAO.getInstance().getJdbcTemp().update(connection -> {
+            PreparedStatement ps = connection.prepareStatement(SP_INSERT, Statement.RETURN_GENERATED_KEYS);
+            new ArgumentPreparedStatementSetter(new Object[]{
+                    entity.getXid(),
+                    entity.getName(),
+                    entity.getVectorImage(),
+                    entity.getComponentData()
+            }).setValues(ps);
+            return ps;
+        }, keyHolder);
+        entity.setId(keyHolder.getKey().intValue());
+        return entity;
+    }
+
+    @Override
+    public List<ScadaObjectIdentifier> getBriefList() {
+        return DAO.getInstance().getJdbcTemp()
+                .query(
+                        ScadaObjectIdRowMapper.selectScadaObjectIdFrom(TABLE_NAME),
+                        new ScadaObjectIdRowMapper()
+                );
+    }
+
+    @Override
+    public List<SynopticPanel> getAll() {
+        return DAO.getInstance().getJdbcTemp()
+                .query(SP_SELECT_LIST, new SynopticPanelRowMapper());
+    }
+
+    @Override
+    public SynopticPanel getById(int id) throws EmptyResultDataAccessException {
+        return DAO.getInstance().getJdbcTemp()
+                .queryForObject(
+                        SP_SELECT + " where " + COLUMN_NAME_SP_ID + "=?",
+                        new Object[]{id},
+                        new SynopticPanelRowMapper()
+                );
+    }
+
+    @Override
+    public Object update(SynopticPanel entity) {
+        DAO.getInstance().getJdbcTemp()
+                .update(
+                        SP_UPDATE,
+                        entity.getXid(),
+                        entity.getName(),
+                        entity.getVectorImage(),
+                        entity.getComponentData(),
+                        entity.getId());
+        return getById(entity.getId());
```

Comment:
You can catch an exception and return a boolean to be sure the update is successful. I don't see any point in doing an update and getting the object right away, since our object represents its state, we don't have to get it from the database.